pkg/server/container_create_test.go

```diff
@@ -1230,3 +1230,78 @@ func TestDisableCgroup(t *testing.T) {
 	t.Log("cgroup path should be empty")
 	assert.Empty(t, spec.Linux.CgroupsPath)
 }
+
+func Test_generateUserString(t *testing.T) {
```

Comment:
nit: TestGenerateUserString to follow the convention in the file? :)

Reply:
The func is un-exported `generateUserString` and it turns out that go test doesnt find `TestgenerateUserString` because the first letter after `Test` is lowercase. So the only way to have it match is:
1. Name it as you did by changing the case of the func we are testing (which seems odd since its not exported).
2. Insert the `_` after the `Test` keyword. 

Not tied to either let me know what you prefer.

Reply:
I think Kubernetes's convention is `TestGernerateUserString`. :)

For example: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/eviction/helpers.go#L102
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/eviction/helpers_test.go#L43

Reply:
@jterry75 I also can't find `Test_` in containerd/containerd, let's follow the Kubernetes convention then.

Otherwise, ready to merge. :)